src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -45,33 +50,61 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
   const focusRef = useAutoFocus<HTMLInputElement>(focus);
   const hasErrors = typeof errors !== 'undefined';
 
-  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
     const changedVal = e.currentTarget.value;
     setTouched?.(true);
     onChangeHandler(changedVal);
     onValidateHandler?.(setErrors, changedVal);
   };
 
+  const handleClickShowPassword = () => {
+    setShowPassword(!showPassword);
+  };
+
+  const handleMouseDownPassword = (e: ClickEvent) => {
+    e.preventDefault();
+  };
+
+  const isConfirmPassword = name === 'confirmPassword';
+
   return (
     <Box>
-      <PasswordInput
-        label={label + (required ? ' *' : '')}
-        tooltipLabel={
-          (isHidden: boolean) => (isHidden ? getTranslation(translations, 'show') : getTranslation(translations, 'hide'))
-        }
-        value={value}
-        name={name}
+      <InputLabel htmlFor={useOid(name)}>{label + (required ? ' *' : '')}</InputLabel>
```

Comment:
is `useOid` necessary here? I think we can (and should) just set it to `name` since the ID of the input element is set to `name`

Reply:
This was transferred from the odyssey component, upon further inspection it looks like its not necessary